Sources/Workspace/Workspace.swift

```diff
@@ -329,21 +343,66 @@ public class Workspace {
     ///
     /// The root package path is used to compute the build directory and other
     /// default paths.
-    public static func create(
+    public convenience init(
+        forRootPackage packagePath: AbsolutePath,
+        toolchain: UserToolchain? = nil,
+        repositoryManager: RepositoryManager? = nil,
+        delegate: WorkspaceDelegate? = nil
+    ) throws {
+        // 👀 is this correct (default toolchain)
```

Comment:
👀 @neonichu @abertelrud does this seem correct?

Reply:
Seems reasonable to me.  Or is this a case where we should require a toolchain to be specified?